### ISSUE TITLE
Support attribute strings that span multiple lines

### ISSUE THREAD

**Issue by timiyay** [ember-template-lint/ember-template-lint#579]
I've been experimenting with an EmberMap-inspired style of stretching `class` strings over multiple lines (from the Tailwind CSS series). In particular, this helps with applying dynamic classes, using inline if statements.

For example:

```hbs
<my-component
  class="flex text-white
  {{if (eq bgColor 'red') 'bg-red'}}
  "
>
```

Note that the `class` string is opened on line 2 and closed on line 4.

This usage fails in the template linter, due to `attribute-indentation`. It expects the closing bracket to be on line 3.

If I put the `class` on a single line, including the if statement, it fails the `attribute-indentation` rule for different reasons:

```
Incorrect indentation of positional param 'eq'...
Incorrect indentation of close curly braces
```

Is this a situation we might want to support?

Keeping these dynamic classes in the template is something I like, in contrast to putting them in a computed property.

It also lets me group Tailwind classes on multiple lines for readability:

```html
<div
  class="flex items-center justify-between
  max-w-xs sm:max-w-sm md:max-w-md
  "
>
```

This issue is possibly related / duplicating https://github.com/ember-template-lint/ember-template-lint/issues/549.

**Comment by lifeart** [comment]
Closing issue, but it's still listed here: https://github.com/ember-template-lint/ember-template-lint/issues/3209
